project/Dependencies.scala

```diff
@@ -18,52 +18,33 @@ import sbt._
 
 object Version {
   val geotools    = "23.2"
-  val spire       = "0.13.0"
+  val spire       = "0.17.0-M1"
```

Comment:
Spark 3 upgrades the breeze dependency as well as the transitive spire dependency.

Reply:
I'm wondering, is it compatible with `0.17.0`? (tests pass though)

Reply:
Lets file CQs for both version and explore how much it would hurt us in terms of https://github.com/locationtech/geotrellis/issues/3380